libs/application/templates/official-journal-of-iceland/src/screens/InvolvedPartyScreen.tsx

```
@@ -0,0 +1,107 @@
+  const { involvedParties, error, loading } = useInvolvedParties({
+    applicationId: application.id,
+    onError: () => {
+      setSubmitButtonDisabled && setSubmitButtonDisabled(true)
+    },
+    onComplete: (data) => {
+      const involvedParties =
+        data?.officialJournalOfIcelandApplicationGetUserInvolvedParties
+          .involvedParties
+
+      if (involvedParties.length === 0 || involvedParties.length > 1) {
+        setSubmitButtonDisabled && setSubmitButtonDisabled(true)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)